src/pipeline-server/.env

```diff
@@ -13,4 +13,26 @@ APPLICATION_RESTART_FREQ_SECS=60
 RUNTIME_LICENSE_CHECK_FREQ_SECS=3600
 http_proxy=
 https_proxy=
-RTSP_CAMERA_IP=
\ No newline at end of file
+RTSP_CAMERA_IP=
+
+# Proxy settings (if applicable)
+http_proxy=http://your_proxy_url
+https_proxy=http://your_proxy_url
+no_proxy=localhost,127.0.0.1,.example.com
```

Comment:
The proxy ENV are pulled directly from the host system so you don't need to define them in the env file.